ui/analyse/src/treeView/common.ts

```diff
@@ -100,15 +100,46 @@ export function findCurrentPath(c: AnalyseCtrl): Tree.Path | undefined {
   );
 }
 
-export function renderInlineCommentsOf(ctx: Ctx, node: Tree.Node): MaybeVNodes {
+export const truncatedComment = <A>(a: A, toHtml: (a: A) => string, path: string | undefined, ctx: Ctx): Hooks => ({
+  insert(vnode: VNode) {
+    (vnode.elm as HTMLElement).addEventListener('click', () => {
+      // Switch to the associated move, or to the root if path is undefined
+      ctx.ctrl.userJumpIfCan(path || '');
+      // Select the comments tab in the underboard - this is wrong
+      ctx.ctrl.study?.vm.toolTab('comments');
+      //Redraw everything
+      ctx.ctrl.redraw();
+      // Scroll down to the comments tab
+      $('.analyse__underboard')[0]?.scrollIntoView();
+    });
+
+    (vnode.elm as HTMLElement).innerHTML = toHtml(a);
+    vnode.data!.cachedA = a;
+  },
+  postpatch(old: VNode, vnode: VNode) {
+    if (old.data!.cachedA !== a) {
+      (vnode.elm as HTMLElement).innerHTML = toHtml(a);
+    }
+    vnode.data!.cachedA = a;
+  },
```

Comment:
looks like low level code pasted from `innerHTML`? Why not use that function instead?

Reply:
The most recent commit to this branch should fix this issue, as long as it's fine to add an additional parent selector to truncated comments.